t/job-manager/plugins/dependency-test.c

```diff
@@ -0,0 +1,90 @@
+/************************************************************\
+ * Copyright 2021 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+/* dependency-test.c - keep jobs in depend state and wait for
+ *  an RPC to release
+ */
+
+#include <jansson.h>
+
+#include <flux/core.h>
+#include <flux/jobtap.h>
+
+
+static void remove_cb (flux_t *h,
+                       flux_msg_handler_t *mh,
+                       const flux_msg_t *msg,
+                       void *arg)
+{
+    flux_jobid_t id;
+    const char *name = NULL;
+    flux_plugin_t *p = arg;
+
+    if (flux_request_unpack (msg, NULL,
+                             "{s:I s:s}",
+                             "id", &id,
+                             "name", &name) < 0) {
```

Comment:
should name be description for consistency? not necessary, sort of borderline, i know this is a test

Reply:
yeah, does seem arbitrary since there's no requirement that plugin protocol match the what is defined in the eventlog, but to avoid confusion I'll rename it.